eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -484,11 +484,22 @@
           "justification": "Removing Jackson annotations from Tables in transition to stream-style."
         },
         {
-          "regex": true,
```

Comment:
why modify this?

Reply:
Merge `revapi` configuration for `attributeRemoved` and `attributeAdded`.